### DISCUSSION TITLE
Whats causing this bug on scores like this?

### DISCUSSION THREAD

**Discussion by localcake** [ppy/osu#21300] [Q&A] [answered]
![nice](https://user-images.githubusercontent.com/88810043/202733792-587e5f26-6493-4aea-9bd6-026b3a7e14ea.png)
![Screenshot 2022-11-12 100252](https://user-images.githubusercontent.com/88810043/202733835-0298e90b-5833-4572-8ad6-26739b1a0e50.png)

This issue may have already been addressed but im just confused

**Comment by frenzibyte** [comment]
I'm failing to see what the "bug" is, can you elaborate further? All I'm seeing here are two completely different scores and one uses regular DT while other uses DT customised to 1.05x speed rate.

**Reply by ghost** [reply]
they are presumably cheated scores

**Reply by frenzibyte** [reply]
Please explain further, the score cards don't even show any signs of being cheated (though I'm not really sure what are the beatmaps of those scores to begin with).

**Reply by localcake** [reply]
I called this a bug, but I'm actually not sure what happened with these scores. The top score is impossible and the bottom score doesn't seem realistic if you look at the pp value.

**Reply by ghost** [reply]
there is a bug with customized dt values showing incorrect pp values, see #21125

**Comment by frenzibyte** [comment, marked as answer]
There can be multitude of reasons on how this could've happened (from faking client hash to modifying the beatmap before server-side hash checks were in place), and trying it can only put you at risk. Report any score you find suspicious via profile.

**Comment by Flamiii** [comment]
PercyDan's play is on an edited version of the map that has 2617 circles and no sliders. You can tell because there are no sections for slider ticks or slider ends on the results screen. This used to be possible up until a few months ago as lazer couldn't detect whether you had locally made changes to a map, but now lazer handles this properly in a similar way to stable so this isn't possible anymore. 

ShaneLiang's play is set on a custom DT speed multiplier of 1.05x and has a broken pp value because lazer has some weird issues with calculating custom mod settings and pp. His play is being treated as if it was normal DT (1.50x speed), hence why it is 1222pp. It should actually be ~495pp (which you can see in the performance breakdown), but his play is completely legitimate and can just be attributed to lazer being a bit weird with pp.

You can report suspicious players like this if you want, but personally I don't do it and I would advise against it because plays like this are usually more of a result of people messing around with lazer as opposed to people actually using any sort of cheats. Leaderboards weren't even added to lazer at the time that both of these scores were set (they were added in July this year), so it's not like they set these scores to get to the top of the map's leaderboard. That being said, if you see weird scores like PercyDan's score that were set a bit more recently, it may not be a bad idea to report the player in question as it's much more likely to be attributed to cheats as opposed to any sort of bugs.
